src/components/AppShell.jsx

```diff
@@ -0,0 +1,129 @@
+import {
+  AppShell,
+  Burger,
+  Footer,
+  Header,
+  MediaQuery,
+  Menu,
+  Navbar,
+  Stack,
+  Text,
+  UnstyledButton,
+  useMantineTheme,
+} from "@mantine/core";
+import {
+  IconLogout,
+  IconMoon,
+  IconSettings,
+  IconSunFilled,
+  IconSwitchHorizontal,
+  IconUserCircle,
+} from "@tabler/icons-react";
+import { useState } from "react";
+import { DemoComponent } from "./DemoComponent/DemoComponent";
+
+export default function AppShellDemo() {
```

Comment:
1) I don't think we need a `default` export here
2) Rename the component from `AppShellDemo` to `AppShell`

Reply:
Good point!

Reply:
Ooops - name collision with Mantine's AppShell component 🙂

Reply:
Ok, just rename it to ApplicationShell

Reply:
> I don't think we need a `default` export here

Yes, we do 🙂

![image](https://user-images.githubusercontent.com/68731150/225576327-9ea4d9fb-4447-443b-a45b-befbc9e551ba.png)

Reply:
Did you change an import at `App.jsx`? It should work

Reply:
I get it, already changed!